http/tokenV2/middleware.go

```diff
@@ -138,18 +177,95 @@ func credentialIsSecure(credential string) error {
 	return fmt.Errorf("no signatures found")
 }
 
+// bestPracticesCheck ensures tokens are crafted in a sensible way, ensuring that even a valid signer must
+// conform to certain security controls such as not creating long lived credentials etc.
+//
+// WARNING: This check does not verify signatures or even the essential claims such as NotBefore, Expiration, etc.
+func bestPracticesCheck(token jwt.Token) error {
+	// Ensure the IssuedAt (iat) field is present
+	if _, ok := token.Get(jwt.IssuedAtKey); !ok {
+		return fmt.Errorf("missing field: iat")
+	}
+
+	// Ensure the Expiration (exp) field is present
+	if _, ok := token.Get(jwt.ExpirationKey); !ok {
+		return fmt.Errorf("missing field: exp")
+	}
+
+	// Ensure the NotBefore (nbf) field is present
+	if _, ok := token.Get(jwt.NotBeforeKey); !ok {
+		return fmt.Errorf("missing field: nbf")
+	}
+
+	// Ensure the expiration is no more than 24.5 hours after NotBefore
+	maxExpirationAfterNotBefore := token.NotBefore().Add(time.Minute * time.Duration(1470))
```

Comment:
Why not just 24h?

Reply:
I should probably leave a better comment, but it's to allow skew and avoid an annoying problem where something is 24.0001 hours etc.